core/src/main/java/com/orange/ouds/core/component/OudsControl.kt

```diff
@@ -20,13 +20,13 @@ import com.orange.ouds.core.utilities.getPreviewEnumEntry
 import com.orange.ouds.foundation.extensions.orElse
 
 internal enum class OudsControlState {
-    Enabled, Hovered, Pressed, Disabled, Focused;
+    Enabled, Hovered, Pressed, Disabled, Focused, ReadOnly;
```

Comment:
I don't know why we have this order. In Figma, state order is: `Enabled`, `Hover`, `Focused`, `Pressed`, `ReadOnly` and `Disabled`. I'm tempted to keep Figma order but that means that we must regenerate all snapshots, and I am not sure it is worth it.